modules/kubernetes/opa-gatekeeper/templates/gatekeeper-values.yaml.tpl

```diff
@@ -1,7 +1,6 @@
 postInstall:
   labelNamespace:
     enabled: false
-experimentalEnableMutation: true
```

Comment:
is this the new default value?

Reply:
Yes it has been removed for another value, mutations are now enabled by default.